README.md

```diff
@@ -898,6 +900,78 @@ Data type: `String`
 
 Name. Made up of access_list and the entry with an underscore seperator. eg. list42_10 is from access_list list42 and entry 10.
 
+### ios_ntp_access_group
+
+Configures NTP Access Group for network devices.
+
+#### Properties
+
+The following properties are available in the `ios_ntp_access_group` type.
+
+##### `name`
+
+Data type: `String`
+
+Configuration name, ip access list name
+
+##### `access_group_type`
+
+Data type: `Enum["peer", "serve", "query-only", "serve-only"]`
+
+Defines the access group type
+
+##### `ipv6_access_group`
+
+Data type: `Optional[Boolean]`
+
+Whether this access group makes use of ipv6
+
+##### `ensure`
+
+Data type: `Enum[present, absent]`
+
+Whether this access group should be present or absent on the target system.
+
+Default value: present
+
+### ios_ntp_config
+
+Extends the ntp_config type.
+
+#### Properties
+
+The following properties are available in the `ios_ntp_config` type.
+
+##### `name`
+
+Data type: String
+
+Config name, default to "default" as the NTP config is global rather than instance based
+
+##### `authenticate`
+
+Data type: Optional[Boolean]
+
+NTP authentication enabled [true|false]
+
+##### `source_interface`
+
+Data type: Optional[String]
+
+The source interface for the NTP system
+
+##### `trusted_key`
+
+Data type: Optional[Array[Variant[Integer, String]]]
+
+Array of global trusted-keys. Contents can be a String or Integers
+
+##### `update_calendar`
```

Comment:
So this is safe to mark as the only available command for this type as the others are an extension of the standard `ntp_config` type, and can just refer them to it; similar to: https://github.com/puppetlabs/cisco_ios#attributes-1 where it has `See radius_global for other available fields`